packages/webview_flutter/webview_flutter_wkwebview/example/integration_test/webview_flutter_test.dart

```diff
@@ -48,30 +46,7 @@ Future<void> main() async {
   final String secondaryUrl = '$prefixUrl/secondary.txt';
   final String headersUrl = '$prefixUrl/headers';
 
-  testWidgets('initialUrl', (WidgetTester tester) async {
-    final Completer<WebViewController> controllerCompleter =
-        Completer<WebViewController>();
-    final Completer<void> pageFinishedCompleter = Completer<void>();
-    await tester.pumpWidget(
-      Directionality(
-        textDirection: TextDirection.ltr,
-        child: WebView(
-          key: GlobalKey(),
-          initialUrl: primaryUrl,
-          onWebViewCreated: (WebViewController controller) {
-            controllerCompleter.complete(controller);
-          },
-          onPageFinished: pageFinishedCompleter.complete,
-        ),
-      ),
-    );
-
-    final WebViewController controller = await controllerCompleter.future;
-    await pageFinishedCompleter.future;
-
-    final String? currentUrl = await controller.currentUrl();
-    expect(currentUrl, primaryUrl);
-  });
+  WebViewPlatform.instance = WebKitWebViewPlatform();
```

Comment:
Shouldn't we be making this fresh for each test in a `setUp` (or locally in the tests) to avoid the possibility of cross-test state contamination?

Reply:
This line is actually no longer necessary since the auto registration was added. I just forgot to remove it. It's now removed, thanks for catching this.